packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/TreeSitterScopeHandler/BaseTreeSitterScopeHandler.ts

```diff
@@ -67,9 +69,18 @@ export abstract class BaseTreeSitterScopeHandler extends BaseScopeHandler {
               targets.length > 1 &&
               !equivalentScopes.every((scope) => scope.allowMultiple)
             ) {
-              throw Error(
-                "Please use #allow-multiple! predicate in your query to allow multiple matches for this scope type",
+              const message =
+                "Please use #allow-multiple! predicate in your query to allow multiple matches for this scope type";
+
+              showError(
+                ide().messages,
+                "BaseTreeSitterScopeHandler.allow-multiple",
+                message,
               );
+
+              if (ide().runMode === "test") {
```

Comment:
Development is much easier if you can see the scopes that allow-multiple is complaining about. If you fail, you can't see anything so it's really hard to debug